src/Timeline.js

```diff
@@ -330,6 +330,7 @@ export default class Timeline extends Axis {
 
     this._brushGroup = elem("g.brushGroup", {parent: this._group});
     this._brushGroup.call(brush).transition(this._transition)
+      .attr("class", this._buttonBehaviorCurrent === "buttons" ? "d3plus-timeline-buttons" : "d3plus-timeline-ticks")
```

Comment:
This logic does not work on re-renders, as you are changing the class from "brushGroup" to "d3plus-timeline-*", so that on re-render the selection for "g.brushGroup" will not work and it will create another group.

1. Set the class to either "brushGroup buttons" or "brushGroup ticks".
2. Move this attr setting before the previous transition (there can be funky behavior when transitioning a property that cannot be transitioned, like "class").